src/utils/SiriusAdapter.cpp

```diff
@@ -209,7 +209,7 @@ class TOPPSiriusAdapter :
     const QString isotope = getStringOption_("isotope").toQString();
     const QString noise = QString::number(getIntOption_("noise"));
     const QString ppm_max = QString::number(getIntOption_("ppm_max"));
-    const QString candidates = QString::number(getIntOption_("candidates"));
+    const QString candidates = QString::number(getIntOption_("candidates")+1);
```

Comment:
better no to use a string here but a Size type.
Conversion to string can be done if required in line 311